pixart.js

```diff
@@ -1 +1,37 @@
+$(document).ready(function(){
 
+var buttonClicked = $("#set-color");
+
+function changeColor(evt){
+  var colorSelected=$("#color-field").val();
+  evt.preventDefault();
+  $("div .brush").css("background-color",colorSelected);
+  console.log("color changed to "+ colorSelected);
+  $("body").on("click",squareCreated);
+}
+
+buttonClicked.on("click",changeColor);
+
+$("#color-field").on("keypress",enterKeyPressed);
```

Comment:
Looking into how you could use the `.on("submit", ...)` listener on `<form>`. Might help make your code a bit cleaner...